dev-python/vdf/vdf-4.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..13} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="A module for (de)serialization to and from VDF, Valve's key-value text format"
+HOMEPAGE="https://github.com/solsticegamestudios/vdf"
```

Comment:
Please add here also the pypi homepage: https://pypi.org/project/vdf/

Reply:
I dropped the pypi homepage since this one still references the old/abandoned repository. Should I still add it?

Reply:
> I dropped the pypi homepage since this one still references the old/abandoned repository. Should I still add it?

I'd keep it for now, the pypi project might be reclaimed by the new upstream in the future.

Reply:
Fixed!